routing/localchans/manager_test.go

```diff
@@ -35,6 +40,18 @@ func TestManager(t *testing.T) {
 		channelSet         []channel
 	)
 
+	sp := [33]byte{}
+	_, _ = hex.Decode(sp[:], []byte("028d7500dd4c12685d1f568b4c2b5048e85"+
```

Comment:
Nit: I would remove the `_, _` and just call the function ?

Reply:
The linter complained that I was ignoring the result when I did that lol

Reply:
ok yes let's keep it then.